src/sentry/identity/oauth2.py

```diff
@@ -76,6 +80,12 @@ def get_oauth_client_secret(self):
     def get_oauth_scopes(self):
         return self.config.get('oauth_scopes', self.oauth_scopes)
 
+    def get_oauth_refresh_token(self):
+        return self.config.get('refresh_token')
```

Comment:
This seems a little strange, why would the refresh token be something that comes from the configuration object that's passed to the provider?

I'm not sure if this is used anywhere, maybe it's just some code that didn't get removed?

Reply:
Good catch. I'll remove it.